testlib/build.gradle

```diff
@@ -13,6 +13,5 @@ dependencies {
 }
 
 // we'll hold the testlib to a low standard (prize brevity)
-findbugs {
-	reportLevel = 'high' // low|medium|high (low = sensitive to even minor mistakes)
+findbugs { reportLevel = 'high' // low|medium|high (low = sensitive to even minor mistakes)
 }
```

Comment:
It's not clear to me why Greclipse formatted this section like this.

Can we prevent it with a setting?